.travis.yml

```diff
@@ -10,6 +10,7 @@ script:
 dist: trusty
 sudo: required
 env:
+  - TEST_SUITE=haml
```

Comment:
rspec and the old test suite.

Would you be fine if I added this to the Rubocop test suite? :thinking: